modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/ParameterModelsLoaderDelegate.java

```diff
@@ -154,14 +151,6 @@ private void parseConfigOverride(ExtensionParameter extensionParameter, Paramete
     }
   }
 
-  private void parseDefaultEncoding(ExtensionParameter extensionParameter, ParameterDeclarer parameter) {
-    // TODO: MULE-9220 - Add a syntax validator which checks that the annotated parameter is a String
```

Comment:
shouldn't delete this TODO. It should actually move the ModelValidator which verifies you didn't use `@DefaultEncoding` more than once